src/libs/ReactNativeLauncher.js

```diff
@@ -222,7 +222,8 @@ class ReactNativeLauncher extends Launcher {
       if (ensureResult.createdJob) {
         this.emit('CREATED_JOB', ensureResult.createdJob)
       }
-      await this.sendLoginSuccess()
+
+      launcherEvent.emit('loginSuccess', ensureResult.createdAccount?._id)
```

Comment:
why are we sending the `createdAccount._id`? It seems that we don't read it from the other side (cause we read it from the flow)? https://github.com/cozy/cozy-libs/blob/88ea4b202b6766db64f1cd22cd4c7d9412d7f8e0/packages/cozy-harvest-lib/src/components/FlowProvider.jsx#L132-L153

Reply:
Actually, not reading it on harvest side was my mistake, because sometimes, the "loginSuccess" event can get to harvest before the account and trigger creation realtime events.
harvest is now fixed and uses this account id